data/templates/pppoe/ipv6-up.script.tmpl

```diff
@@ -36,6 +36,11 @@ echo 1 > /proc/sys/net/ipv6/conf/{{ ifname }}/forwarding
 #
 echo 2 > /proc/sys/net/ipv6/conf/{{ ifname }}/accept_ra
 
+{% if default_route == 'none' %}
+# Prevent learning of default router from router advertisements
+echo 0 > /proc/sys/net/ipv6/conf/{{ ifname }}/accept_ra_defrtr
```

Comment:
If I switch back to `default_route auto`, where is this case handled?

Reply:
If `accept_ra_defrtr` is not set, the functional default is to follow what `accept_ra` is set to (in the sense that if `accept_ra` is >= 1, `accept_ra_defrtr` will also be 1).

Since the PPPoE interface disappears when it goes down, we don't have to worry about any 'old' value staying around. I have tested this with setting 'default-route' to none, committing, and the followed by `default-route` to auto, and the routing table correctly gets the default route.

Reply:
Please explicitly enable it so it's easier to read/understand. Other then that it looks good to me. THanks.

Reply:
I've updated the template to explicitly enable `accept_ra_defrtr` when `default-route` is not none.